src/main/resources/application.yaml

```diff
@@ -32,32 +32,12 @@ spring:
       - a60-form
       - c100-transform
       - a58-form
+      - A59-form
```

Comment:
Can we follow what is existing naming standards

Reply:
Done

Reply:
Corrected file names